common-fragment-V1/pom.xml

```diff
@@ -548,6 +548,11 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.pentaho</groupId>
+            <artifactId>pentaho-hadoop-shims-common-services-api</artifactId>
```

Comment:
Isn't this kind of introducing a circular dependency? Doesn't the `pentaho-hadoop-shims-common-services-api` depend on this module? 
The `pentaho-hadoop-shils-common-services-api` depends on `DriverLocatorImpl` that is declared in this module.
https://github.com/pentaho/pentaho-hadoop-shims/blob/82c0a8d19c2058dda50abdb3d19cd3f5cb1028c3/common-services-api/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L38
https://github.com/pentaho/pentaho-hadoop-shims/blob/master/common-fragment-V1/src/main/java/org/pentaho/big/data/api/jdbc/impl/DriverLocatorImpl.java

Reply:
No. The dependency of pentaho-hadoop-shims-common-services-api on common-fragment-V1 is removed in these set of changes. Now only the common-fragment-V1 dependency on pentaho-hadoop-shims-common-services-api will remain.